requirements.txt

```diff
@@ -1,18 +1,28 @@
+# Core
 click>=7.1
 colorama>=0.3
 configparser
-cached-property
-# oyaml is like pyyaml but preserves orderings
```

Comment:
Any reason for removing this comment?

Reply:
@tunetheweb Was just because it wasn't in the setup.py comments and did a straight swap. I've re-added it now 👍

Reply:
On a related question should we order these alphabetically to make it easier to find if a dependency is in this file?

Reply:
👀

Reply:
@tunetheweb I've reordered the deps alphabetically in requirements.txt and setup.py. I left the ordering as is in requirements_dev.txt as I think the current grouping is clearer